cmd/influxd/upgrade/upgrade.go

```diff
@@ -258,7 +259,7 @@ func NewCommand() *cobra.Command {
 		},
 	}
 
-	cli.BindOptions(cmd, opts)
+	cli.BindOptions(v, cmd, opts)
 	// add sub commands
 	cmd.AddCommand(v1DumpMetaCommand)
 	cmd.AddCommand(v2DumpMetaCommand)
```

Comment:
Do these 2 subcommands need the viper for anything?

Reply:
No, they should probably be removed actually.